addon/components/polaris-badge.js

```diff
@@ -65,66 +60,61 @@ export default class PolarisBadge extends Component {
    * @default null
    * @public
    */
-  progress = null;
+  progress;
 
   /**
    * Medium or small size. Use `small` only in the main navigation of an app frame..
    *
    * @type {String}
-   * @default 'medium'
+   * @default null
    * @public
    */
-  size = DEFAULT_SIZE;
+  size;
```

Comment:
any reason we removed the default? it seems it's still the case [here](https://github.com/Shopify/polaris-react/blob/master/src/components/Badge/Badge.tsx#L49)

Reply:
Nice job with `argWithDefault` decorator 👏 

I've seen this other format in [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap/blob/master/addon/components/bs-button.js#L186) which I've also found nice,  closer to plain JS and love the short name. Interested to hear what you think cc @andrewpye 
ex:
```js
@arg
size = DEFAULT_SIZE
```

Reply:
I also like the short `@arg` syntax for its brevity and for keeping the default value "inline" with the prop declaration (sorry @tomnez 😝)

Reply:
Ha I'm good with this too. I liked the `WithDefault` part of the name though, I know `@arg` is nice and short but it isn't really descriptive as to why it's being used in some places but not others for named arguments 🤷